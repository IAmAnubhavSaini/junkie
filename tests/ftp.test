#!/bin/sh

NAME="FTP"
PCAPS="pcap/ftp/ftp_port_cmd.pcap"
filter() {
	grep '^FTP' | cut -d' ' -f 2-
}
OUTPUT="ftp.out"

. ./dump_test
