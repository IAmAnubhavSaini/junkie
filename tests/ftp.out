-- pcap/ftp/ftp_port_cmd.pcap --
head_len=0, payload=20
head_len=0, payload=16
head_len=0, payload=34
head_len=0, payload=21
head_len=0, payload=23
head_len=0, payload=14
head_len=0, payload=26
head_len=0, payload=7
head_len=0, payload=37
head_len=0, payload=8
head_len=0, payload=30
head_len=0, payload=27
head_len=0, payload=51
head_len=0, payload=11
head_len=0, payload=61
head_len=0, payload=9
head_len=0, payload=24
